developer/prog-guide/creating-a-windows-powershell-navigation-provider.md

```diff
@@ -13,57 +13,43 @@ ms.assetid: 8bd3224d-ca6f-4640-9464-cb4d9f4e13b1
 caps.latest.revision: 5
 ---
 # Creating a Windows PowerShell Navigation Provider
-This topic describes how to create a Windows PowerShell navigation provider that can navigate the data store. This type of provider supports recursive commands, nested containers, and relative paths.
 
-> [!NOTE]
->  You can download the C# source file (AccessDBSampleProvider05.cs) for this provider using the Microsoft Windows Software Development Kit for Windows Vista and .NET Framework 3.0 Runtime Components. For download instructions, see [How to Install Windows PowerShell and Download the Windows PowerShell SDK](http://msdn.microsoft.com/en-us/3ef7402e-fc80-432d-aaf7-c4a43fc09e68).
```

Comment:
What happened to this content?

Reply:
This was trimmed out by my code and I re-ran it against the old version and fixed it.